### PR TITLE
chore(sdk): Fix a bug where we missed injecting importer node

### PR DESCRIPTION
**Pull request by chensun**
**Description of your changes:**
importer spec may get its `artifact_uri` value from constant value passed directly into component factory function.
(see changes made to `input_5` in sdk/python/kfp/v2/compiler_cli_tests/test_data/pipeline_with_various_io_types.py)

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention). 

- [ ] Do you want this pull request (PR) cherry-picked into the current release branch?

    [Learn more about cherry-picking updates into the release branch](https://github.com/kubeflow/pipelines/blob/master/RELEASE.md#cherry-picking-pull-requests-to-release-branch).

### CODE REVIEW

**Review by numerology — COMMENTED** [review]
Thanks!

File: sdk/python/kfp/v2/dsl/component_bridge.py [hunk 1]
```diff
@@ -50,6 +51,8 @@ def create_container_op_from_component_and_arguments(
   # might need to append suffix to exuector_label to ensure its uniqueness?
   pipeline_task_spec.executor_label = component_spec.name
 
+  importer_spec = {}
```
[the remark this answers is not in the document]
Comment (chensun) [reply sdk/python/kfp/v2/dsl/component_bridge.py:55]: Doen.

File: sdk/python/kfp/v2/dsl/importer_node.py [hunk 2]
```diff
@@ -13,31 +13,58 @@
...
+    pipeline_param_name: str = None,
+    constant_value: str = None
+) -> pipeline_spec_pb2.PipelineDeploymentConfig.ImporterSpec:
+  """Build an importer executor spec.
+
+  Args:
+    input_type_schema: The type of the input artifact.
+    pipeline_param_name: The name of the pipeline parameter if the importer gets
+      its artifacts_uri via a pipeline parameter.
+    constant_value: The value of artifact_uri in case a contant value is passed
```
[the remark this answers is not in the document]
Comment (chensun) [reply sdk/python/kfp/v2/dsl/importer_node.py:33]: Done.

**Review by chensun — COMMENTED** [review]
Thanks for reviewing.

**Comment by numerology** [comment]
/lgtm
/approve

Thanks!

**Pull request merged**